src/main/java/com/cs56fitnessapp/models/Meal.java

```diff
@@ -1,29 +1,137 @@
 package com.cs56fitnessapp.models;
+import java.util.ArrayList;
+
 
 /**
- * @author Nerissa Hsieh
+ * @author Nerissa Hsieh & Jimwell Castillo
  * Date Created: 10/03/17
  * Date Modified: 10/03/17
  */
 
-public class Meal implements DietFacts {
-    @Override
-    public int getCaloriesIn() {
-        return 0;
-    }
+public class Meal implements DietFacts
+    {
```

Comment:
Please use either indent K&R style (preferably) or Allman ident style. 
GNU is standing a lot from the rest of the code we have - it's not a good practice. 
[reference](https://en.wikipedia.org/wiki/Indentation_style)